library/item/item_setting/src/main/java/org/quiltmc/qsl/item/setting/mixin/recipe_remainder/AbstractFurnaceBlockEntityMixin.java

```diff
@@ -106,8 +130,16 @@ private static void setInputRemainder(ItemStack inputStack, int amount, DynamicR
 				recipe,
 				inventory,
 				INPUT_SLOT,
-				quilt$THREAD_LOCAL_BLOCK_ENTITY.get().getWorld(),
-				quilt$THREAD_LOCAL_BLOCK_ENTITY.get().getPos()
+				remainder -> {
```

Comment:
Im guessing that this works with the new mixin method introduced. I would add a comment explaining this since its a bit confusing about how this prevents a NPE.